pkg/model/types.go

```diff
@@ -119,9 +183,9 @@ func (e *Endpoint) String() string {
 	return string(bytes)
 }
 
-// EndpointIdFromIPAddress converts an IP address to human readable identifier.
-func EndpointIdFromIPAddress(address string) string {
-	return strings.Replace(address, ".", "_", -1)
+// EndpointIdFromIPAddress converts an IP address to human-readable identifier.
+func EndpointIdFromIPAddress(address string, port int32) string {
+	return fmt.Sprintf("%s:%d", address, port)
```

Comment:
I was amazed to see that periods and colons are valid in instance identifiers, since the Cloud Map console rejects it. We should let them know.